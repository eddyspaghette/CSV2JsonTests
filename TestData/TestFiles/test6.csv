ッ,あ,が,ざ,℁,⏳
ツ,ぃ,ぎ,せ,℟,▓
ヅ,い,け,₠,Å,○
ッ,あ,が,ざ,℁,⏳
ド,え,さ,℀,ↂ,🍕
ト,ぇ,ご,₣,ψ,😵
ド,え,さ,℀,ↂ,🍕
