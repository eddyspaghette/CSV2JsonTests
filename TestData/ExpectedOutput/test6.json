[
 {
   "FIELD1": "„ÉÉ",
   "FIELD2": "„ÅÇ",
   "FIELD3": "„Åå",
   "FIELD4": "„Åñ",
   "FIELD5": "‚ÑÅ",
   "FIELD6": "‚è≥"
 },
 {
   "FIELD1": "„ÉÑ",
   "FIELD2": "„ÅÉ",
   "FIELD3": "„Åé",
   "FIELD4": "„Åõ",
   "FIELD5": "‚Ñü",
   "FIELD6": "‚ñì"
 },
 {
   "FIELD1": "„ÉÖ",
   "FIELD2": "„ÅÑ",
   "FIELD3": "„Åë",
   "FIELD4": "‚Ç†",
   "FIELD5": "‚Ñ´",
   "FIELD6": "‚óã"
 },
 {
   "FIELD1": "„ÉÉ",
   "FIELD2": "„ÅÇ",
   "FIELD3": "„Åå",
   "FIELD4": "„Åñ",
   "FIELD5": "‚ÑÅ",
   "FIELD6": "‚è≥"
 },
 {
   "FIELD1": "„Éâ",
   "FIELD2": "„Åà",
   "FIELD3": "„Åï",
   "FIELD4": "‚ÑÄ",
   "FIELD5": "‚ÜÇ",
   "FIELD6": "üçï"
 },
 {
   "FIELD1": "„Éà",
   "FIELD2": "„Åá",
   "FIELD3": "„Åî",
   "FIELD4": "‚Ç£",
   "FIELD5": "œà",
   "FIELD6": "üòµ"
 },
 {
   "FIELD1": "„Éâ",
   "FIELD2": "„Åà",
   "FIELD3": "„Åï",
   "FIELD4": "‚ÑÄ",
   "FIELD5": "‚ÜÇ",
   "FIELD6": "üçï"
 }
]